pymc/distributions/continuous.py

```diff
@@ -2531,6 +2531,16 @@ def logcdf(value, alpha, beta):
             msg="alpha > 0, beta > 0",
         )
 
+    def icdf(value, alpha, beta):
+        res = 1 / Gamma.icdf(value=1 - value, alpha=alpha, scale=1 / beta)
```

Comment:
For curiosity any intuition why the icdf looks like it does, specially the flipped value. It's interesting